File: benchmark.py
# Python Speed Benchmark Test
# Running 100 Million Iterations in a Loop

import time

print("Python Benchmark: Starting 100,000,000 iterations...")
start_time = time.time()

i = 0
count = 0
while i < 100000000:
    count += 1
    i += 1

end_time = time.time()
elapsed = end_time - start_time

print("Python Benchmark Finished!")
print(f"Final Count: {count}")
print(f"Time taken by Python: {elapsed:.4f} seconds")

File: benchmark.js
// JavaScript Speed Benchmark Test
// Running 100 Million Iterations in a Loop

console.log("JavaScript (Node.js) Benchmark: Starting 2,000,000,000 iterations...");
const start = performance.now();

let i = 0;
let count = 0;
while (i < 2000000000) {
    count++;
    i++;
}

const end = performance.now();
const elapsed = (end - start) / 1000;

console.log("JavaScript Benchmark Finished!");
console.log(`Final Count: ${count}`);
console.log(`Time taken by JavaScript: ${elapsed.toFixed(4)} seconds`);
